content/en/guides/core/registry/configure_registry.md

```diff
@@ -117,11 +153,29 @@ For example, in the proceeding image Alex inherits **Member** role privileges be
 {{< img src="/images/registry/role_conflict.png" alt="Registry role conflict resolution" >}}
 
 
-## Configure registry roles
-1. Navigate to the Registry at https://wandb.ai/registry/.
-2. Select the registry you want to configure.
-3. Click the gear icon on the upper right hand corner.
-4. Scroll to the **Registry members and roles** section.
-5. Within the **Member** field, search for the user or team you want to edit permissions for.
-6. In the **Registry role** column, click the user's role. 
-7. From the dropdown, select the role you want to assign to the user.
\ No newline at end of file
+### SDK compatibility
+
+{{% alert title="SDK version requirement" %}}
+To use the W&B SDK to access artifacts as a **Restricted Viewer**, you must use W&B SDK version 0.19.9 or higher. Otherwise, SDK commands will result in permission errors.
+{{% /alert %}}
+
+When a **Restricted Viewer** uses the SDK, certain functions are not available or work differently.
+
+- **`use_artifact()`**: Not available. Results in permission errors.
+- **`artifact.download()`**: Not available. Results in permission errors.
+- **`artifact.files()`**: Available for viewing artifact directory structure and artifact metadata only.
+- **`artifact.file()`**: Available for viewing artifact metadata only.
+- **`artifact.get()`**: Available for viewing artifact metadata only.
+- **`artifact.get_path()`**: Available for viewing artifact metadata only.
+- **`artifact.get_entry()`**: Available for viewing artifact metadata only.
+- **`artifact.json_encode()`**: Available for viewing artifact metadata only.
+- **`artifact.verify()`**: Available for viewing artifact metadata only.
```

Comment:
Some of these are purposely hidden. See the Python SDK docs for which should and should not be exposed: https://docs.wandb.ai/ref/python/sdk/classes/artifact/

Reply:
After applying Ishita's feedback I verified and all of the remaining ones are referenced in the SDK docs. PTAL and let me know if this is the correct way to refer to these functions.

Reply:
Thanks. Python Classes are capitalized, so each that start with "artifact" need to be like: `Artifact.download()`

Which `use_artifact()` is this referring to, btw? 
https://docs.wandb.ai/ref/python/sdk/classes/run/#method-runuse_artifact
https://docs.wandb.ai/ref/python/public-api/runs/#method-runuse_artifact

Reply:
Fixed the formatting and turned each into a link so it is clear which method we mean.